src/schema/v2/artist/helpers.ts

```diff
@@ -1,6 +1,6 @@
 import { compact, sortBy } from "lodash"
 import { priceDisplayText } from "lib/moneyHelpers"
-
+import { camelCase } from "lodash"
```

Comment:
I think the import should be `import camelCase from 'lodash/camelCase';` to avoid importing the whole lodash(which is quite big)

Reply:
doesn't increase memory consumption in this case?